HTTSM2017/bin/MorphingSM2017.cpp

```diff
@@ -103,16 +103,29 @@ int main(int argc, char **argv) {
 
   // Define background processes
   map<string, VString> bkg_procs;
-  if (jetfakes) {
-    bkg_procs["et"] = {"ZTT", "ZL", "TTT", "VVT", "EWKZ", "jetFakes"};
-    bkg_procs["mt"] = {"ZTT", "ZL", "TTT", "VVT", "EWKZ", "jetFakes"};
-    bkg_procs["tt"] = {"ZTT", "ZL", "TTT", "VVT", "EWKZ", "jetFakes"};
-    bkg_procs["em"] = {"ZTT", "ZL", "TTT", "VVT", "EWKZ", "jetFakes"};
+  if (jetfakes) { 
```

Comment:
Das ist irgendwie unschoen. Embedding und fake factors modifizeren unabhaengige Prozesse (?) Dh, kann man nicht die full-MC collection von den backgrounds nehmen und je nach Option diese Collection editieren? So was ungefaehr (in C++):

```python
bkgs = ["W", "ZTT", "QCD", "ZL", "ZJ", "TTT", "TTJ", "VVJ", "VVT", "EWKZ"]
if embedding:
    bkgs = [b for b in bkgs if bkgs not in  ["ZTT", "TTT"]] + ["EMB", "TTL"]
if jetfakes:
    bkgs = [b for b in bkgs if bkgs not in ["QCD", "VVJ", "TTJ"]] + ["jetFakes"]
```

Reply:
Am besten drunter noch eine kurze section, die die selektierten Prozesse printed. Sonst hab ich angst, dass wir irgendwo was vergessen.

Reply:
ja, finde ich gut so